cdc/processor/manager.go

```diff
@@ -131,7 +131,9 @@ func (m *Manager) closeProcessor(changefeedID model.ChangeFeedID) {
 	if processor, exist := m.processors[changefeedID]; exist {
 		err := processor.Close()
 		if err != nil {
-			log.Warn("failed to close processor", zap.Error(err))
+			log.Warn("failed to close processor",
+				zap.String("changefeedID", changefeedID),
```

Comment:
```suggestion
				zap.String("changefeed", changefeedID),
```

Prefer `changefeed`.